schema_based_on_diagram.sql

```diff
@@ -0,0 +1,79 @@
+CREATE TABLE "treatments"(
+    "id" INTEGER NOT NULL,
+    "type" VARCHAR(255) NOT NULL,
+    "name" VARCHAR(255) NOT NULL
+);
+ALTER TABLE
+    "treatments" ADD PRIMARY KEY("id");
+CREATE TABLE "medical_histories"(
+    "id" INTEGER NOT NULL,
+    "admitted_at" TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    "patient_id" INTEGER NOT NULL,
+    "status" VARCHAR(255) NOT NULL
+);
+CREATE INDEX "medical_histories_patient_id_index" ON
+    "medical_histories"("patient_id");
+ALTER TABLE
+    "medical_histories" ADD PRIMARY KEY("id");
+CREATE INDEX "medical_histories_patient_id_index" ON
+    "medical_histories"("patient_id");
+CREATE TABLE "Patients"(
+    "id" BIGINT NOT NULL,
+    "name" VARCHAR(255) NOT NULL,
+    "date_of_birth" VARCHAR(255) NOT NULL
+);
+ALTER TABLE
+    "Patients" ADD PRIMARY KEY("id");
+CREATE TABLE "invoice_items"(
+    "id" INTEGER NOT NULL,
+    "unit_price" DECIMAL(8, 2) NOT NULL,
+    "quantity" INTEGER NOT NULL,
+    "total_price" DECIMAL(8, 2) NOT NULL,
+    "invoice_id" INTEGER NOT NULL,
+    "treatment_id" INTEGER NOT NULL
```

Comment:
- [ ] This is a foreign key. kindly consider to change this line to: 
`treatment_id int references treatments(id),` to add the foreign key constraint.